htdocs/mrp/class/mo.class.php

```diff
@@ -735,7 +735,10 @@ public function updateProduction(User $user, $notrigger = true)
 	 */
 	public function delete(User $user, $notrigger = false)
 	{
+		$result = $this->call_trigger('MRP_MO_DELETE', $user);
```

Comment:
A trigger is already automatically included into the deleteCommon() method.
Name is MO_DELETE